### DISCUSSION TITLE
FZ-like CLI over USB serial

### DISCUSSION THREAD

**Discussion by eadmaster** [whatotter/pwnhyve#34] [Ideas]
I think this could be An handy alternative to remote control over wifi+ssh, possibly the rpi could be powered from the same usb cable (no need for a battery):

https://www.reddit.com/r/flipperzero/comments/wepbbg/you_can_use_flipper_on_the_go_on_anroid_with/
https://docs.flipper.net/development/cli

**Comment by whatotter** [comment]
that's actually a good idea, but instead of using serial (very slow), i can do an RNDIS gadget, so ssh and vnc would still be possible over usb - i will work on this over the weekend, and let you know :)

**Reply by eadmaster** [reply]
seems a good alternative to using VNC as discussed here: https://github.com/whatotter/pwnhyve/issues/30 https://github.com/whatotter/pwnhyve/issues/37
